addons/account/models/account_move.py

```diff
@@ -1864,7 +1864,9 @@ def copy(self, default=None):
             default['date'] = self.company_id._get_user_fiscal_lock_date() + timedelta(days=1)
         if self.move_type == 'entry':
             default['partner_id'] = False
-        return super(AccountMove, self).copy(default)
+        copied_am = super(AccountMove, self).copy(default)
```

Comment:
Since you are editing this line already, you could remove useless parameters of `super`

```suggestion
        copied_am = super().copy(default)
```